templates/cross-cutting/region.html.j2

```diff
@@ -46,18 +46,18 @@ Upon completion of all entry activities of <i>{{ render_name_with_icon(tx.source
 {% endif %}
 
 {% if state.entry %}
-<p>On entry into the <b>{{ state.name }}</b> {{ state.__class__.__name__ }} the system shall perform the following activities:
+<p>On entry into the <b>{{ state.name }}</b> {{ state.__class__.__name__ }} the system shall perform the following activities:</p>
 {% for fnc in state.entry %}
 {{ render_name_with_icon(fnc) | safe }};
-{% endfor %}</p>
+{% endfor %}
```

Comment:
I think this was correct before, but now no longer is. `render_name_with_icon` produces `<a>` tags, which need to be inside of a block (like `<p>`), and they were supposed to be rendered on the same line just after the colon.

Reply:
After investigating with @jamilraichouni, we figured that this was probably supposed to be a `<ul>` like all the other instances. Same for the other case below.

Reply:
After looking at it again I think this is supposed to be kind of inline and not as a list.
The semicolon is not given in other cases of the template.

Reply:
I'm fine either way. Let's just keep it like it is now.